distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup.adoc

```diff
@@ -0,0 +1,31 @@
+:title: High Availability Initial Setup
+:type: installing
+:status: published
+:summary: Initial setup in a Highly Available Cluster.
+:project: ${branding}
+:order: 09
+
+=== {title}
+
+This section describes how to complete the initial setup of the ${branding} in a <<_introduction_to_high_availability,Highly Available Cluster>>.
+
+.Prerequisites
+* An HTTP proxy that can route HTTP traffic according to the pattern described in the Introduction to High Availability.
+The open-source proxy that was used for testing was HAProxy.
+(Refer to HAProxy's http://cbonte.github.io/haproxy-dconv/1.7/management.html#4[Management Guide] for setup instructions.)
+* Solr Cloud: See <<_solr_cloud_prerequisites, Solr Cloud section>> for guidance on installation as well
+ as configuring the ${branding}s to connect to Solr Cloud
+
+Once the prerequisites have been met, the below steps can be followed.
+
+[NOTE]
+====
+Unless listed in the <<_high_availability_initial_setup_exceptions,High Availability Initial Setup Exceptions>> section,
+the normal steps can be followed for installing, configuring, and hardening.
+====
+
+. Install the first ${branding} node. See the <<_installing, Installation Section>>.
+. Configure the first ${branding} node. See the <<_configuring, Configuring Section>>.
+. Optional: Harden the first ${branding} node (excluding setting directory permissions). See the <<_security_hardening, Hardening Section>>.
+. Export the first ${branding} node's configurations and import them on the second ${branding} node. See <<_reusing_configurations, Reusing Configurations>>.
```

Comment:
Before being able to import on the second node, it has to be installed.

Reply:
I don't explicitly mention it here because the install step is mentioned in your import documentation.

Reply:
I think we need to mention it here, even if it is slightly redundant. Since this is a high level snapshot of what an admin needs to do.

Reply:
Done